src/Microplate/CoordinateSystem2x16MM.php

```diff
@@ -0,0 +1,20 @@
+<?php declare(strict_types=1);
+
+namespace MLL\Utils\Microplate;
+
+class CoordinateSystem2x16MM extends CoordinateSystem
```

Comment:
"MM" ist irgendwie nicht ausreichend. Es ist ja total komisch, dass das J "fehlt" und das müssen wir irgendwie auch in dem namen Wiederspigeln..

- CoordinateSystemTecan2x16NoJ
- CoordinateSystem2x16SkippedJ

@spawnia Hast du eine Idee wie man das nennen könnte?

Reply:
MM sagt mir auch gar nichts. Finde die Variante mit `NoJ` gut.